src/main/resources/edu/cornell/kfs/fp/cu-ojb-fp.xml

```diff
@@ -1288,4 +1288,38 @@
         <field-descriptor name="documentCount" column="DOC_CNT" jdbc-type="INTEGER" />
     </class-descriptor>
 
+    <class-descriptor class="edu.cornell.kfs.fp.businessobject.TravelMealCardVerificationData" table="CU_FP_TMCARD_VERIFY_T">
+        <field-descriptor name="cardHolderAccountNumber" column="CARD_ACCOUNT_NBR" jdbc-type="VARCHAR" primarykey="true" index="true" />
+        <field-descriptor name="emplid" column="EMPLID" jdbc-type="VARCHAR" primarykey="true" index="true" />
+        <field-descriptor name="cardHolderLine1Address" column="ADDRESS1" jdbc-type="VARCHAR" />
+        <field-descriptor name="cardHolderLine2Address" column="ADDRESS2" jdbc-type="VARCHAR" />
```

Comment:
Is there a reason you decided to name these variables cardHolderLine1Address instead of cardHolderAddressLine1?